### PR TITLE
S02 us5 bdd

### PR DESCRIPTION
**Pull request by Mcbess-san**

### CODE REVIEW

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

File: src/View/Home/_carousel.html.twig [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                <div class="carousel-caption d-flex justify-content-center">
+                    <h3>Clemence.D</h3>
+                        <p class="d-flex justify-content-center">J'ai consulté Mme MILLIET 2 fois en quatre ans et je ne peux que vous la recommander, elle est bienveillante, vous met à l'aise, 
+                            vous explique parfaitement bien comment va se dérouler la séance et on ressort de chez elle véritablement apaisée. Pour ma part en tout cas, 
+                            je suis ravie et toutes ces prédictions et avertissements se sont avérées utiles et justes.
+                        </p>
+                </div>
+            </div>
+            <div class="carousel-item">
+                <img src="/assets/images/592367.jpg" class="d-block w-100 h-100" alt="...">
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:30]: images naming + "..." is not a valid alt attribute

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

File: public/assets/css/testimony.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        font-size: var(--font-xxsmall-size);
+    }
+    .carousel-caption h3 {
+        font-size: var(--font-xxsmall-size);
+    }
+}
+
+@media screen and (max-width: 450px){
+    .carousel-caption p {
+        font-size: 10px;
```
Comment (RaphaelBS-WCS) [inline public/assets/css/testimony.css:46]: magic values

File: public/assets/css/testimony.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    }
+    .carousel-caption h3 {
+        font-size: 10px;
```
Comment (RaphaelBS-WCS) [inline public/assets/css/testimony.css:49]: Magic Values

**Review by RaphaelBS-WCS — CHANGES_REQUESTED** [review]
See Comments

**Pull request closed without merging**